decred_wallet/Features/History/TransactionHistoryViewController.swift

```diff
@@ -57,6 +58,7 @@ class TransactionHistoryViewController: UIViewController {
         }
         self.visible = true
         if (self.FromMenu){
+            refreshControl.programaticallyBeginRefreshing(in: self.tableView)
```

Comment:
Do we only need this loading indicator when this page is accessed from menu?

Reply:
Yes, swipe down refresh handles the loading indicator in the app